recipes/approvaltests.cpp/config.yml

```diff
@@ -3,3 +3,7 @@ versions:
     folder: all
   8.1.0:
     folder: all
+  8.1.2:
```

Comment:
```suggestion
  8.1.1:
```

Reply:
Thanks @uilianries - I've pushed the edit...
Am currently working on scripting the editing of this file for future releases